Sources/Classes/BiometryService.swift

```diff
@@ -7,18 +7,59 @@
 
 import LocalAuthentication
 
+/// The protocol used to define the specifications necessary for the SDK.
```

Comment:
I think `necessary` should be placed before `specifications` since `specifications` is noun and in this context `necessary` is an adjective instead 🙏 
```suggestion
/// The protocol used to define the necessary specifications for the SDK.
```